modules/sc-mesh-secure-deployment/src/2_0/features/jamming/options.py

```diff
@@ -0,0 +1,87 @@
+import os
+import re
+from typing import List
+
+
+VALID_CHANNELS = [36, 40, 44, 48, 149, 153, 157, 161]
+
+
+class Options:
+    def __init__(self):
+        self.jamming_osf_orchestrator: str = 'fd01::1'
+        self.port: int = 8080
+        self.starting_channel: int = 36
+        self.waiting_time: int = 15
+        self.channels5: List[int] = [36, 40, 44, 48, 149, 153, 157, 161]
+        self.threshold: float = 0.0
+        self.osf_interface: str = 'tun0'
+        self.scan_interface: str = 'wlp1s0'
+        self.mesh_interface: str = 'wlp1s0'
```

Comment:
Hardcoded network interface names?

Reply:
The options.py file is the configuration file where the user is expected to manually configure all variables related to the jamming avoidance scheme, which includes setting the interface name used to perform the spectral scan, as well as the name of the mesh interface. All the jamming avoidance code files will get the config information from the options.py file